client/Components/MiddlePackage.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+
+const MiddlePackage = (props) => {
+
+    return (
+
+        <div className='middlePackage'>
+            <span className='quantityTitle'>Quantity</span>
```

Comment:
Css naming convention would be quantity-title. Check it out, gurl:
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/